db/wal_edit_test.cc

```diff
@@ -49,19 +49,6 @@ TEST(WalSet, Overwrite) {
   ASSERT_EQ(wals.GetWals().at(kNumber).GetSyncedSizeInBytes(), kBytes);
 }
 
-TEST(WalSet, SmallerSyncedSize) {
```

Comment:
I think we should keep this test, and assert that the synced WAL size is unchanged.

Reply:
I keep it after merging main-branch to current PR